### PR TITLE
Interactive

### PR DESCRIPTION
**Pull request by mbonabucya**
### add Interaction to the list

- [ ] add interaction to the list
- [ ]  implement completed button
- [ ] Add event listener to the checkbox (change).
- [ ] Update items object's value for completed key upon user actions.
- [ ] Implement a function for the "Clear all completed" button (use filter() method).
- [ ] Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.
interactivity list pr

### CODE REVIEW

File: src/storage.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export const saveStorage = (items) => {
+  localStorage.setItem('todo-list', JSON.stringify(items));
+};
+
+export const getStorage = () => {
+  const tasks = localStorage.getItem('todo-list') ? JSON.parse(localStorage.getItem('todo-list')) : [];
+  return tasks;
+};
```
Comment (roshan-bajgain) [inline src/storage.js:8]: - [ ] The index value should start from 1 but your index value is starting from 2 kindly fix it.
- [ ] The index value is duplicating kindly fix it and when I delete the value the index should update but the index is not updating kindly fix it as well.
### Image
local storage

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @mbonabucya, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/storage.js (hunk 1 above)
Comment (mbonabucya) [reply src/storage.js:8]: @roshan-bajgain 

Thank you for your feedback

Could you please check again . it is working as expected for me
when I add the tasks it starts from the index 1 
when I delete any task it is being updated.
Interactive PR

**Review by Munsa1 — APPROVED** [review]
Hi @mbonabucya, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**